test/functional/apps/settings/_index_pattern_create_delete.js

```diff
@@ -0,0 +1,109 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var SettingsPage = require('../../../support/pages/SettingsPage');
+  var expect = require('intern/dojo/node!expect.js');
+  var Promise = require('bluebird');
+
+  return function (bdd) {
+    bdd.describe('creating and deleting default index', function describeIndexTests() {
+      var common;
+      var settingsPage;
+      var remote;
+
+      bdd.before(function () {
+        common = new Common(this.remote);
+        settingsPage = new SettingsPage(this.remote);
+        remote = this.remote;
+      });
+
+
+      bdd.describe('index pattern creation', function indexPatternCreation() {
+        bdd.beforeEach(function be() {
```

Comment:
Why is the index pattern getting created and deleted after every single test? The tests themselves don't seem to modify any state, so I'd expect a single creation and single deletion would suffice for the entire module which would probably speed things up quite a bit.